src/reactviews/pages/QueryResult/textView.tsx

```diff
@@ -0,0 +1,220 @@
+                        const separatorLine = columnWidths
+                            .map((width) => "-".repeat(width))
+                            .join("  ");
+                        content += separatorLine + "\n";
+
+                        // Add the formatted data rows
+                        for (const formattedRow of formattedRows) {
+                            content += formattedRow + "\n";
+                        }
+
+                        // Add row count information
+                        if (resultSetSummary.rowCount > 0) {
+                            content += `(${resultSetSummary.rowCount} row${resultSetSummary.rowCount === 1 ? "" : "s"} affected)\n`;
+                        } else {
+                            content += "(0 rows affected)\n";
+                        }
+
+                        content += "\n";
+                        resultSetNumber++; // Increment for next result set
+                    }
+                }
+
+                if (content.trim() === "") {
+                    content = "No results to display.";
+                }
+            } catch (error) {
+                console.error("Error generating text view:", error);
+                content = "Error generating text view. Please try switching back to grid view.";
+            }
+
+            setTextContent(content);
+            setLoading(false);
+        };
+
+        void generateTextView();
+    }, [uri, resultSetSummaries, webViewState]);
+
+    if (loading) {
+        return <div className={classes.noResults}>Loading text view...</div>;
+    }
+
+    return (
+        <div className={classes.textViewContainer}>
+            {textContent ? (
+                <Editor
+                    height="100%"
+                    language="plaintext"
+                    theme={resolveVscodeThemeType(context?.themeKind || ColorThemeKind.Light)}
+                    value={textContent}
+                    options={{
+                        readOnly: true,
+                        minimap: { enabled: false },
+                        scrollBeyondLastLine: false,
+                        wordWrap: "off",
+                        fontFamily: fontSettings.fontFamily || "var(--vscode-editor-font-family)",
+                        fontSize: fontSettings.fontSize || 12,
+                        lineNumbers: "off",
+                        glyphMargin: false,
+                        folding: false,
+                        lineDecorationsWidth: 0,
+                        lineNumbersMinChars: 0,
+                        renderLineHighlight: "none",
+                        scrollbar: {
+                            vertical: "auto",
+                            horizontal: "auto",
+                        },
+                        automaticLayout: true,
+                    }}
+                />
+            ) : (
+                <div className={classes.noResults}>No results to display in text format.</div>
```

Comment:
Loc.  But also "No results to display in text format" implies that there might be results to display in grid format, so I don't like that wording.